src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -1782,6 +1844,104 @@ static void refda_adm_ietf_dtnma_agent_ctrl_ensure_odm(refda_ctrl_exec_ctx_t *ct
      * |START CUSTOM FUNCTION refda_adm_ietf_dtnma_agent_ctrl_ensure_odm BODY
      * +-------------------------------------------------------------------------+
      */
+
+    /*
```

Comment:
Just realized that these are in the ADM itself; these can be moved into actual tests under `item-test/test_refda_stdio.py` to exercise real behavior.

Reply:
Done, thanks